### PR TITLE
New testupdate urls for IBS maintenance

### PR DESCRIPTION
**Pull request by dirkmueller**
you.suse.de is going away with the upcoming IBS switch to Build
Service Maintenance.

### CODE REVIEW

**Comment by toabctl** [comment]
+1

**Comment by bmwiedemann** [comment]
It seems, only the SP4 and SLE12 ones are really active yet. Marcus said, SP3 one is still to come.

**Comment by dirkmueller** [comment]
@bmwiedemann splitted the part that can be merged into https://github.com/SUSE-Cloud/automation/pull/412

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -779,10 +795,11 @@ function onadmin_prepare_sles12_cloud_repo()
 
 function onadmin_prepare_sles12_other_repos()
 {
-    for repo in SLES12-{Pool,Updates}; do
-        add_mount "$repo/sle-12-x86_64" "clouddata.cloud.suse.de:/srv/nfs/repos/$repo" \
-            "$tftpboot_repos12_dir/$repo"
-    done
+    add_mount "SLES12-Pool/sle-12-x86_64" "clouddata.cloud.suse.de:/srv/nfs/repos/SLES12-Pool" \
```
Comment (aspiers) [inline scripts/qa_crowbarsetup.sh]: What's the reason for this change?

**Comment by toabctl** [comment]
Anything left todo here? Still +1

**Comment by dirkmueller** [comment]
@toabctl I think a second +1 is missing.. :-)

**Comment by dirkmueller** [comment]
well, jenkins passes, so merging anyway.
